source/mongoexport.txt

```diff
@@ -705,6 +705,13 @@ Options
    You can sort the results with the :option:`--sort` option to
    :binary:`~bin.mongoexport`.
    
+.. option:: --queryFile=<filename>
+
+   An alternative to :option:`--query <mongoexport --query>`. The
+   :option:`--queryFile <mongoexport --query>` option allows you to
+   specify in a file the query in :manual:`Extended JSON v2 format
+   (either relaxed or canonical/strict mode)
```

Comment:
[suggestion] I think we can remove this parenthetical since both options are permitted and users that aren't familiar with Extended JSON can learn about the two formats.
```suggestion

```